components/automate-ui/src/app/components/create-object-modal/create-object-modal.component.html

```diff
@@ -44,6 +44,11 @@ <h2 slot="title">Create {{ objectNoun | titlecase }}</h2>
             <chef-button tertiary (click)="modifyID = true" id="edit-button-object-modal" data-cy="edit-button">Edit ID</chef-button>
           </chef-toolbar>
         </div>
+        <div class="checkbox" *ngIf="projectNoteVisible">
```

Comment:
We want to show a checkbox even though it cannot be unchecked? I wonder if we should leave this off for the moment and just have an italicized note saying that they will be generated. Even though this saves us "future time".